drivers.json

```diff
@@ -854,52 +860,53 @@
-                    "version": "515.105.01",
+                    "version": "525.125.06",
                     "nvenc_patch": true,
                     "nvfbc_patch": true,
-                    "driver_url": "https://international.download.nvidia.com/XFree86/Linux-x86_64/515.105.01/NVIDIA-Linux-x86_64-515.105.01.run"
+                    "driver_url": "https://international.download.nvidia.com/XFree86/Linux-x86_64/525.125.06/NVIDIA-Linux-x86_64-525.125.06.run"
                 },
                 {
-                    "version": "525.116.03",
+                    "version": "525.147.05",
                     "nvenc_patch": true,
                     "nvfbc_patch": true,
-                    "driver_url": "https://international.download.nvidia.com/XFree86/Linux-x86_64/525.116.03/NVIDIA-Linux-x86_64-525.116.03.run"
+                    "driver_url": "https://international.download.nvidia.com/XFree86/Linux-x86_64/525.1047.05/NVIDIA-Linux-x86_64-525.147.05.run"
                 },
                 {
-                    "version": "535.43.02",
+                    "version": "530.30.02",
                     "nvenc_patch": true,
                     "nvfbc_patch": true,
-                    "driver_url": "https://international.download.nvidia.com/XFree86/Linux-x86_64/535.43.02/NVIDIA-Linux-x86_64-535.43.02.run"
+                    "driver_url": "https://international.download.nvidia.com/XFree86/Linux-x86_64/530.30.02/NVIDIA-Linux-x86_64-530.30.02.run"
                 },
                 {
-                    "version": "525.116.04",
+                    "version": "530.41.03",
                     "nvenc_patch": true,
                     "nvfbc_patch": true,
-                    "driver_url": "https://international.download.nvidia.com/XFree86/Linux-x86_64/525.116.04/NVIDIA-Linux-x86_64-525.116.04.run"
+                    "driver_url": "http://international.download.nvidia.com/XFree86/Linux-x86_64/530.41.03/NVIDIA-Linux-x86_64-530.41.03.run"
                 },
                 {
-                    "version": "535.54.03",
+                    "version": "535.43.02",
                     "nvenc_patch": true,
                     "nvfbc_patch": true,
-                    "driver_url": "https://international.download.nvidia.com/XFree86/Linux-x86_64/535.54.03/NVIDIA-Linux-x86_64-535.54.03.run"
+                    "driver_url": "https://international.download.nvidia.com/XFree86/Linux-x86_64/535.43.02/NVIDIA-Linux-x86_64-535.43.02.run"
                 },
+
                 {
-                    "version": "525.125.06",
+                    "version": "535.54.03",
                     "nvenc_patch": true,
                     "nvfbc_patch": true,
-                    "driver_url": "https://international.download.nvidia.com/XFree86/Linux-x86_64/525.125.06/NVIDIA-Linux-x86_64-525.125.06.run"
+                    "driver_url": "https://international.download.nvidia.com/XFree86/Linux-x86_64/535.54.03/NVIDIA-Linux-x86_64-535.54.03.run"
```

Comment:
Please revert these manual reorders, I understand the intent of keeping it alphabetically sorted but given this is not a user facing file, and the readme is kept sorted by the readme-autogen, I don't think this is necessary.

Use the `tools/readme-autogen/add_driver.py` script to update this file instead of manual edits.

Reply:
done.